src/dataflow/flow-graph.ts

```diff
@@ -100,7 +101,16 @@ export class FlowGraph {
           throw new Error(`Unknown check result: ${result}`);
```

Comment:
Shouldn't this one be an assert? It can only fail if the code has returned something it shouldn't, ever.

Reply:
Good point, done!